package.json

```diff
@@ -4,10 +4,11 @@
   "description": "[F]ind [R]eal [E]states [d]amn eas[y].",
   "scripts": {
     "prepare": "husky",
-    "start": "node prod.js",
-    "dev": "yarn && rm -rf ./ui/public/* && vite",
```

Comment:
We should assume the project dependencies to be installed before a script is executed:

```json
{
    "dev": "rm -rf ./ui/public/* && vite",
    "ui": "rm -rf ./ui/public/* && vite",
    "prod": "vite build --emptyOutDir",
}
```

Deletion of files inside `ui/public` can be handled via Vite's `emptyOutDir` option:

```json
{
    "dev": "vite --emptyOutDir",
    "ui": "vite --emptyOutDir",
    "prod": "vite build --emptyOutDir",
}
```

After setting the `emptyOutDir` option via `vite.config.js`:

```json
{
    "dev": "vite",
    "ui": "vite",
    "prod": "vite build",
}
```

Then after removing the duplicate script and renaming everything properly:

```json
{
    "start:frontend:dev": "vite",
    "build:frontend": "vite build"
}
```